RELEASENOTES-1.4.md

```diff
@@ -356,3 +356,4 @@
   The bugfix is opt-in, because an immediate bugfix would risk breaking existing builds; the error will only be reported when the flag `--no-compat=broken_unused_types` is passed to DMLC. This flag will be automatically enabled in Simics 8.
 - `release 7 7063`
 - `release 6 6362`
+- `note 6` Corrected read_bits/write_bits -> read_field/write_field in the reference manual.
```

Comment:
A releasenote is redundant, but I personally don't find it redundant _enough_ to protest against it. I will however protest against the formatting. You should tick the references to the identifiers (both out of principle and because the `_` will incur italics otherwise).

Also, the ASCII arrow is slightly questionable (I've no idea how pretty it'll render.) You could go with a unicode arrow, or reformulate to not use the arrow:
> Corrected old references to `read_bits`/`write_bits` in the reference manual to instead reference their modern replacements `read_field`/`write_field`.


... or you could choose to drop the releasenote entirely.